bindings/python/src/alert.cpp

```diff
@@ -441,7 +463,7 @@ void bind_alert()
 
     class_<listen_succeeded_alert, bases<alert>, noncopyable>(
         "listen_succeeded_alert", no_init)
-        .def_readonly("endpoint", &listen_succeeded_alert::endpoint)
```

Comment:
did the ``def_readonly`` not work? (if so, what error did you experience?). Or do you need to be able to write to this field?